src/nvim/highlight.c

```diff
@@ -830,7 +830,8 @@ Dictionary hl_get_attr_by_id(Integer attr_id, Boolean rgb, Arena *arena, Error *
 /// @param[in/out] hl Dictionary with pre-allocated space for HLATTRS_DICT_SIZE elements
 /// @param[in] aep data to convert
 /// @param use_rgb use 'gui*' settings if true, else resorts to 'cterm*'
-void hlattrs2dict(Dictionary *dict, HlAttrs ae, bool use_rgb)
+/// @param nvim_get_hl function is being used by nvim_get_hl
```

Comment:
can that be made more descriptive? i.e., what changes in behavior when this flag is set?

Reply:
How about `color_abbr`?

Reply:
I will change it to `short_keys` 👍🏻